src/msgpack.light/TokenBuilders/IMsgPackTokenBuilder.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace ProGaudi.MsgPack.Light.TokenBuilders
+{
+    internal interface IMsgPackTokenBuilder
+    {
+        void Add(DataTypes dataType);
+
+        MsgPackToken CreateToken(DataTypes dataType);
```

Comment:
As we discussed, MsgPackTokenConverter.Read method is very similar to BaseMsgPackReader.Skip method.
The only difference is that In Read method we collect bytes to buffer and made some operations with it (.ToArray() for example),  but in Skip method we don't do that.
To avoid logic duplication, IMsgPackTokenBuilder introduced.